src/components/organisms/CommentList/index.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react';
+import Styled from 'styled-components';
+import CommentWrite from 'components/molecules/CommentWrite';
+import SingleComment from 'components/molecules/SingleComment';
+
+interface IData {
+  childrenComment: {
+    _id: string;
+    userID: {
+      _id: string;
+      nickname: string;
+    };
+    text: string;
+  }[];
+  _id: string;
+  userID: {
+    _id: string;
+    nickname: string;
+  };
+  text: string;
+}
+
+interface INewComment {
+  _id: string;
+  userID: {
+    _id: string;
+    nickname: string;
+  };
+  text: string;
+}
+
+interface IProps {
+  commentList: Array<IData> | undefined;
```

Comment:
```suggestion
  commentList?: IData[];
```
여기에 이렇게 해주면 더 간결하겠죠?